manifests/init.pp

```diff
@@ -76,6 +76,13 @@
     } ->
   anchor { 'php::end': }
 
+  # Configure global PHP settings in php.ini
+  Anchor['php::begin'] ->
+  class {'::php::global':
+    settings => $real_settings,
+  } ->
+  Anchor ['php::end']
```

Comment:
@minorOffense The whitespace is causing the specs to fail using future parser. :see_no_evil: